src/opening-period/page/EditOpeningPeriodPage.test.tsx

```diff
@@ -0,0 +1,332 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { render, screen } from '@testing-library/react';
+import {
+  DatePeriod,
+  DatePeriodOptions,
+  Resource,
+  ResourceState,
+} from '../../common/lib/types';
+import api from '../../common/utils/api/api';
+import EditOpeningPeriodPage from './EditOpeningPeriodPage';
+
+const testDatePeriodOptions: DatePeriodOptions = {
+  actions: {
+    POST: {
+      resource_state: {
+        choices: [
+          {
+            value: 'open',
+            display_name: 'Open',
+          },
+          {
+            value: 'closed',
+            display_name: {
+              fi: 'Suljettu',
+              sv: null,
+              en: null,
+            },
+          },
+          {
+            value: 'self_service',
+            display_name: 'Self service',
+          },
+        ],
+      },
+    },
+  },
+};
+
+const testResource: Resource = {
+  id: 1186,
+  name: {
+    fi: 'Test resource name in finnish',
+    sv: 'Test resource name in swedish',
+    en: 'Test resource name in english',
+  },
+  address: {
+    fi: 'Test address in finnish',
+    sv: 'Test address in swedish',
+    en: 'Test address in english',
+  },
+  description: {
+    fi: 'Test description in finnish',
+    sv: 'Test description in swedish',
+    en: 'Test description in english',
+  },
+  extra_data: {
+    citizen_url: 'kansalaisen puolen url',
+    admin_url: 'admin puolen url',
+  },
+};
+
+const testDatePeriod: DatePeriod = {
+  id: 1,
+  created: '2020-11-20',
+  modified: '2020-11-20',
+  is_removed: false,
```

Comment:
I can't decide if it's a good idea to add attributes to these that we do not use in the app yet? On the other hand it serves as a good documentation what the object returned by api actually looks. But on the other hand the reader might think that everything he sees here has some purpose and usage, and he is exposed to a complexity he would not need to when he reads these.

Reply:
Removed unused values